docs/tutorials/timeseries/forecasting-faq.md

```diff
@@ -1,42 +1,40 @@
 # AutoGluon Time Series FAQ
 
+## What forecasting tasks can AutoGluon be used for?
+AutoGluon can generate **probabilistic** multi-step-ahead forecasts for one or multiple **univariate** time series.
+For example, you can use AutoGluon to forecast daily sales of multiple products over the next month.
 
-## Where can I find more information about the models/metrics?
+AutoGluon also supports additional information, such as time-independent static features (e.g., location of the store)
+and time-dependent covariates (e.g., price of the product each day).
+See the [In Depth Tutorial](forecasting-indepth.ipynb) for more details.
 
-Metrics are implemented in the `autogluon.timeseries.evaluator` module. We also follow some of
-the same conventions followed by GluonTS in their evaluation.
-Please refer to
-the GluonTS [documentation](https://ts.gluon.ai/stable/api/gluonts/gluonts.html) and
-[github](https://github.com/awslabs/gluon-ts) for further information.
+Currently, AutoGluon does not support features such as hierarchical forecasting and forecast explainability, but we will consider adding them in the future.
 
-A detailed description of evaluation metrics is also available at
-[here](https://docs.aws.amazon.com/forecast/latest/dg/metrics.html).
+## How can I get the most accurate forecasts?
+To maximize the forecast accuracy, set the `predictor.fit()` argument `presets="best_quality"` or `presets="high_quality"` and provide a high `time_limit`.
```

Comment:
We should just mention `best_quality` which maximize the accuracy using all possible models with ensembles unless `high_quality` has the same implementation